dist/bin/startup.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+HOME_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && cd .. && pwd)"
+BIN_DIR=${HOME_DIR}/bin
+LOG_DIR=${HOME_DIR}/logs
+CONFIG_DIR=${HOME_DIR}/configs
+LOG_FILE_LOCATION=${LOG_DIR}/satellite.log
+
+if [ ! -d "${LOG_DIR}" ]; then
+    mkdir -p "${LOG_DIR}"
+fi
+
+if uname -s | grep Darwin; then
+  START_UP_PROCESS=$(find "$BIN_DIR" -name "skywalking-satellite*darwin*")
+elif uname -s | grep Linux; then
+  START_UP_PROCESS=$(find "$BIN_DIR" -name "skywalking-satellite*linux*")
+else
+  echo "Sorry, the official startup process does not support your system."
```

Comment:
You can use `START_UP_PROCESS=$(find "$BIN_DIR" -name "skywalking-satellite*windows*")` here, Windows users can still use this shell script if they use something like Git Bash or Cygwin, otherwise this line of code will never be reached IMO, because Powershell or Command Prompt won't execute `.sh` file at all